recipes/state-machine-cat/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016-2022 Sander Verweij
```

Comment:
```suggestion
Copyright (c) 2016-2023 Sander Verweij
```

(NTS: probably needs  to be adapted in the state-machine-cat license file  proper as  well.